spec/export/index.md

```diff
@@ -267,27 +268,30 @@ Content-Type: application/json
 - HTTP status code of ```5XX```
 - ```Content-Type``` header of ```application/json```
 - The server SHALL return a FHIR OperationOutcome resource in JSON format
-- The choice of when to determine that an export job has failed in its entirety (error status) vs returning a partial success (complete status) is left up to the implementer.
 
 	Example OperationOutcome response body:
 
   ```
+  Status: 500 Internal Server Error
+  Content-Type: application/json
+  Retry-After: 120
```

Comment:
The failed example shouldn't have a retry-after header -- once failed, the request is done. Is there a reason we're including an example here when there's already an example in the table above? Would be better to cut this, unless it's distinct.

Reply:
Yup i agree, we should remove it. I just committed that here https://github.com/HL7/bulk-data/pull/32/commits/72b3f10261b3495259313fe1b9237e8044526215